### PR TITLE
Added RUNTIME cypher option

### PR DESCRIPTION
**Pull request by lutovich**
Possible options are INTERPRETED and COMPILED. Only INTERPRETED is currently supported.

### CODE REVIEW

**Comment by davidegrohmann** [comment]
This is very likely conflicting with https://github.com/neo4j/neo4j/pull/4305. I would suggest to sync before merging.

File: community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/CypherCompiler.scala [hunk 1]
```diff
@@ -162,6 +169,29 @@ class CypherCompiler(graph: GraphDatabaseService,
     if (planner.isEmpty) defaultPlanner else planner.head
   }
 
+  private def parseRuntime(options: Seq[CypherOption], planner: PlannerName, version: CypherVersion) = {
```
Comment (davidegrohmann) [inline community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/CypherCompiler.scala]: `parseRuntime` -> `calculateRuntime` like `calculatePlanner`

**Comment by davidegrohmann** [comment]
@lutovich can we hold on merging this until we have at least one compiled operator to execute?

**Comment by Mats-SX** [comment]
Superseded by #4317

**Pull request closed without merging**